content/integrations/sumologic.md

```diff
@@ -13,11 +13,12 @@ Enable the Sumo Logic integration to:
 
 ## Installation
 
-There are two parts to the SumoLogic integration, based on the direction of information between the products. You can send alerts and events from Datadog to Sumologic and you can have SumoLogic send data to Datadog.
+Datadog integration with SumoLogic is two-part: send alerts and events from Datadog to Sumologic, or have SumoLogic send data to Datadog. You can enable one or both.
```

Comment:
this feels a bit more awkward than the original.  'there are two parts' vs 'xxx is two-part'